server/src/main/java/org/opensearch/plugins/ClusterPlugin.java

```diff
@@ -89,4 +90,13 @@ default Map<String, ExistingShardsAllocator> getExistingShardsAllocators() {
      */
     default void onNodeStarted() {}
```

Comment:
What do you think of marking the `onNodeStarted()` as deprecated so developers will know which one to use?

Reply:
Sounds like a great idea.